.github/ISSUE_TEMPLATE/bug_report.yaml

```diff
@@ -0,0 +1,135 @@
+name: Bug Report
+description: Create a report to help us reproduce and correct the bug
+labels: ['bug']
+
+body:
+- type: markdown
+  attributes:
+    value: >
+      #### Before submitting a bug, please make sure the issue hasn't been already
+      addressed by searching through [the past issues](https://github.com/man-group/ArcticDB/issues).
+
+- type: textarea
+  attributes:
+    label: Describe the bug
+    description: >
+      A clear and concise description of what the bug is.
+  validations:
+    required: true
+
+- type: textarea
+  attributes:
+    label: Steps/Code to Reproduce
+    description: |
+      Please add a [minimal code example](https://matthewrocklin.com/minimal-bug-reports) that can reproduce the error when running it.
+    Be as succinct as possible, **do not depend on external data files**: instead you can generate synthetic data using `numpy.random`.
```

Comment:
How about adding:

```If your bug report appears to be dependant on the external data files and cannot be reproduced without, please describe your data files in as much detail as you can.```

Reply:
s/dependant/dependent
s/on the external/on external
s/without/without them